pyracing/response_objects/session_data.py

```diff
@@ -227,9 +227,9 @@ class RaceLapsDriver:
     def __init__(self, dict):
         self.drivers = [self.Driver(x) for x in dict['drivers']]
         self.header = self.Header(dict['header'])
-        self.lap_data = [self.LapData(x) for x in dict['lapData']]
+        self.lap_data = [self.Lap(x) for x in dict['lapData']]
```

Comment:
If we want the base class to just be `lap` then this field should be `laps` instead of `lap_data`

Reply:
Changed lap_data to laps